root/static/scripts/edit/components/UrlRelationshipCreditFieldset.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import FieldErrors from '../../../../components/FieldErrors';
+import FormRowText from '../../../../components/FormRowText';
+
+export type ActionT = {+credit: string, +type: 'update-relationship-credit'};
+
+type PropsT = {
+  +dispatch: (ActionT) => void,
+  +field: ReadOnlyFieldT<string | null>,
+};
+
+export type StateT = DatePeriodFieldT;
+
+export type WritableStateT = WritableDatePeriodFieldT;
+
+const UrlRelationshipCreditFieldset = ({
+  dispatch,
+  field,
+}: PropsT): React$Element<'fieldset'> => {
+  function handleCreditChange(
+    event: SyntheticKeyboardEvent<HTMLInputElement>,
+  ) {
+    dispatch({
+      credit: event.currentTarget.value,
+      type: 'update-relationship-credit',
+    });
+  }
+
+  return (
+    <fieldset>
+      <legend>{l('Credit')}</legend>
+      <FormRowText
+        field={field}
+        label={addColonText(l('Credited as'))}
```

Comment:
This seems a little confusing because it's inside a dialog for the URL relationship type but there's no context of which entity is being credited.

 * Can we set a placeholder attribute on the input with the source entity name, like the relationship editor dialogs do?
 * What do you think about renaming the field to "Credited to:" instead?  That seems clearer to me because the dialog is for the URL, so it's who the URL is credited to.

Reply:
When editing URL relationships, in the same popup as dates + ended.

Reply:
I changed that item to "Credited to". Did not add a placeholder for now since we no longer do that for credits in the new relationship editor.